package.json

```diff
@@ -105,7 +107,7 @@
     "html-webpack-plugin": "^5.5.0",
     "husky": "^7.0.4",
     "jest": "^27.5.1",
-    "jsdom": "^19.0.0",
+    "jsdom": "^21.1.1",
```

Comment:
This is something I tried, the tests still work with the upgrade. But we might want to revert back so wanted to call it out.